File: Programmation/11_09_20/jours.rb
#!/usr/bin/env ruby

myJours = ["lundi", "mardi", "mercredi", "jeudi", "vendredi", "samedi", "dimanche" ]
myDays = [ "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday" ]
myTage = [ "Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag", "Sonntag" ]

puts "Il y a " + myJours.length.to_s + " jours dans la semaine\n"
 
puts "Le dernier jour de la semaine est : " + myJours.last


puts "Le mot " + myJours[0] + " se traduit par " + myDays[0] + " en anglais et par " + myTage[0] + " en allemand\n\n"

for i in 0..(myJours.length - 1)
  puts "Le mot " + myJours[i].to_s + " se traduit par " + myDays[i].to_s + " en anglais et par " + myTage[i].to_s + " en allemand\n"
end

File: Programmation/TD2/jours.pl
#!/usr/bin/perl -w

use strict ;

my @jours = ( "lundi", "mardi", "mercredi", "jeudi", "vendredi", "samedi", "dimanche" ) ;
my @days = ( "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday" ) ;
my @tage = ( "Montag", "Dienstag", "Mittwoch", "Donnerstag", "Freitag", "Samstag", "Sonntag" ) ;

my $total = @jours ;
my $order ;
print "Il y a $total jours dans la semaine.\n" ;
print "Le dernier jour de la semaine est : $jours[-1]\n\n" ;

print "Affichage des traductions de \@jours en anglais et en allemand :\n" ;
foreach $order(0 .. $#jours){
    print "Le mot \"$jours[$order]\" se traduit par \"$days[$order]\" en anglais et par \"$tage[$order]\" en allemand\n" ;
}
